src/by/it_academy_homework2/tasks/HW2.java

```diff
@@ -0,0 +1,29 @@
+package by.it_academy_homework2.tasks;
+
+import java.util.Scanner;
+
+public class HW2 {
+    public static void main(String[] args) {
+        //найти вещественные корни квадратного уравнения ax^2+bx+c=0 либо сообщить , что их нет
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter the a, b, c: ");
+        int a = scanner.nextInt();
+        int b = scanner.nextInt();
+        int c = scanner.nextInt();
+        printRootsOfEquation(a, b, c);
+    }
+
+    public static void printRootsOfEquation(int a, int b, int c) {
+        int discrim = b * b - 4 * a * c;
+        if (discrim < 0) {
+            System.out.println("Quadratic equation has no roots");
+        } else if (discrim == 0) {
+            double x = -(double) b / (2 * a);
+            System.out.println("Quadratic equation has one root: " + x);
+        } else if (discrim > 0) {
```

Comment:
```suggestion
        } else {
```
(предложение от intellij idea - `Condition 'discrim > 0' is always 'true'`)